### DISCUSSION TITLE
ACR Reactive BluetoothLE Plugin & Shiny.Beacons?

### DISCUSSION THREAD

**Discussion by bodeg** [shinyorg/shiny#688] [Q&A] [answered]
I have Xamarin.Forms app that uses the "good old"  ACR Reactive BluetoothLE Plugin

I want to start shinying my app.
Can I use both  ACR Reactive BluetoothLE Plugin and Shiny.Beacons in the same app?

Thanks, for the great framework.
alon

**Comment by aritchie** [comment, marked as answer]
They should work fine, but the plugin is external to Shiny so it wouldn't be supported if they don't
